prac_06/guitars.py

```diff
@@ -0,0 +1,33 @@
+from prac_06.guitar import Guitar
+
+guitars = []
+
+
+def main():
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitars.append(Guitar(name, year, cost))
+        print(f"{Guitar(name, year, cost)} added.", end="\n\n")
+        name = input("Name: ")
+
+    print("\n...snip...\n")
+    print("These are my guitars:")
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:
```

Comment:
You can use the code that the instruction gives us.

Reply:
Got it, thanks for the comment! ^^